sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -485,23 +489,19 @@ private Read() {}
       final boolean validate;
 
       Bound(Class<T> type) {
-        this(null, null, "", 0, ShardNameTemplate.INDEX_OF_MAX, type, null, true);
+        this(null, FileNameTemplate.DEFAULT, 0, type, null, true);
```

Comment:
Per our discussion, I thought you were going to ditch FileNameTemplate. It only applies to FileBasedSink IIRC and it will only show up in that metadata.

(If not, it should still be in a separate PR. This PR should be "pure" and only add DisplayData.)